development/http-client.http

```diff
@@ -1,20 +1,19 @@
-GET {{url}}/starter.zip
+GET {{url}}/starter.zip?tapirVersion=1.0.0-RC1
 Accept: application/octet-stream
 Content-Type: application/json
 
 {
-  "tapirVersion" : "1.0.0",
-  "scalaVersion" : "2.13",
-  "sbtVersion" : "1.6.2",
-  "projectDetails" : {
-    "group" : "com.softwaremill",
-    "artifact" : "library",
-    "projectName" : "projectName",
-    "packageName" : "com.softwaremill"
+  "projectName" : "projectname",
+  "groupId" : "softwaremill.com",
+  "effect" : {
+    "FutureEffect" : {
+
+    }
   },
-  "moduleDependencies" : {
-    "dependencies" : [
-    ]
+  "implementation" : {
+    "Akka" : {
```

Comment:
hm this looks weird; we should probably pick a different strategy for serialising enum-like fields

Reply:
done